renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
Effectively, we have an older version of the CONST_Makfile, commit like it and I will change it when I do an upgrade.